src/test/java/madalinapopescu/tests/tema7/CuraHealthcareLoghinTests.java

```diff
@@ -0,0 +1,50 @@
+package madalinapopescu.tests.tema7;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+public class CuraHealthcareLoghinTests extends BaseTest{
```

Comment:
Can you please rename the class to CuraHealthcareLoginTests? Be careful to update testng .xml files with suite/groups too. Thanks!